samples/highcharts/demo/bullet-graph/demo.js

```diff
@@ -30,6 +30,12 @@ Highcharts.setOptions({
     }
 });
 
+const customColors = {
+    high: 'rgb(128, 128, 128, 0.1)',
+    med: 'rgb(128, 128, 128, 0.2)',
+    low: 'rgb(128, 128, 128, 0.3)'
+};
```

Comment:
I think we should put this inline again in order to keep the config JSON-compatible and free of variables